apps/frontend/src/libs/components/sidebar/sidebar-item/type.ts

```diff
@@ -0,0 +1,13 @@
+import { type ValueOf } from "@git-fit/shared";
+
+import { type IconName } from "~/libs/components/icon/types/types.js";
+import { type AppRoute } from "~/libs/enums/app-route.enum.js";
+
+type sidebarItemType = {
```

Comment:
Types should start with a capital letter, also they shouldn't include type word itself 
```suggestion
type NavigationItem = {
```